.github/workflows/deploy-production-faq.yml

```
@@ -61,10 +61,25 @@ jobs:
           NG_USE_OFFERS: 'false'
           NG_USE_Q_AND_AS: 'true'
           NG_USE_Q_AND_A_SEARCH: 'true'
-          TXT_APP_NAME: 'FAQ / Vraagbaak'
+          NG_USE_FEEDBACK_PROMPT: 'false'
+          TXT_APP_NAME: 'FAQ \n Vraagbaak'
           TXT_APP_LOGO_URL: 'https://rodekruis.github.io/510-ICON-LIBRARY/Beeldmerk_van_het_Nederlandse_Rode_Kruis.svg'
```

```suggestion
          TXT_APP_LOGO_URL: ''
```

NLRC-logo only in footer instead.